hugo/assets/scss/components/sidenote.scss

```diff
@@ -15,7 +15,7 @@
     font-weight: $weight-medium;
     height: var(--sidenote-height);
     line-height: var(--sidenote-height);
-    margin: 0 0 .5em;
+    margin: 0 0 0.5em;
```

Comment:
is there a reason we use em here instead of rem? rem depend on the base font size (16px in this case). Em use the font-size of the current element as base

Reply:
Just checked since I did this part of the styling I think: we can this to rem because we use rem everywhere. It won't affect the styling here.